### ISSUE TITLE
 No notifications

### ISSUE THREAD

**Issue by spectre4094** [Fahmula/emby-telegram-notifier#19]
Hi,

The only notification I have is the test notification.

However, I chose the addition of media.

Emby Server : 4.80.5.0

![image](https://github.com/Fahmula/emby-telegram-notifier/assets/17724374/e714e83b-1bd6-4e36-bc03-6398d067c8d4)

Thx

**Comment by DickenSerm** [comment]
Have you read through that #16  ?

**Comment by spectre4094** [comment]
![image](https://github.com/Fahmula/emby-telegram-notifier/assets/17724374/533f1050-1f67-45e2-9ad9-b76a0b7e963c)

**Comment by DickenSerm** [comment]
The error message “Max. URL Retries Exceeded" indicates that the request library attempted to connect to the server multiple times and failed.

Is `"http://192.168.2.240:8096"` your EMBY_BASE_URL in the .env file?

**Comment by spectre4094** [comment]
Yes, this is indeed the address of the emby server in the .env file

Test notification works.

![image](https://github.com/Fahmula/emby-telegram-notifier/assets/17724374/57f90a39-dc80-4121-95f6-9802037c66c1)

**Comment by DickenSerm** [comment]
Is the Python script on the same server? Same IP?

**Comment by spectre4094** [comment]
the server is a Synology DS1621+.

Emby is installed via the package center

**Comment by DickenSerm** [comment]
I also have it on a Synology (running DS918+).

Have you tried docker-compose.yml yet?
      `- PUID=${PUID}`
      `- PGID=${PGID}`
to enter in environment?

The values ​​then logically also need to be adjusted in the .env.

If you don't know the values, you can read how to display them [here](https://mariushosting.com/synology-find-uid-userid-and-gid-groupid-in-5-seconds/)

**Comment by spectre4094** [comment]
Deployment problem by Portenair. 

![image](https://github.com/Fahmula/emby-telegram-notifier/assets/17724374/fd0de803-c351-47e4-8a25-4f8c2db82647)

**Comment by DickenSerm** [comment]
In Portainer you can create the variables below the web editor open. Without. Env file:

![image-png-Feb-21-2022-06-25-09-63-PM.png](https://github.com/Fahmula/emby-telegram-notifier/assets/77905751/e6eabc05-f642-4edd-bdb0-3e59e07c19e1)

[Imagesource](https://www.portainer.io/blog/using-env-files-in-stacks-with-portainer?hs_amp=true)

**Comment by spectre4094** [comment]
![2024-04-29_162215](https://github.com/Fahmula/emby-telegram-notifier/assets/17724374/3e485cb9-53b6-4085-b712-3d01fed60b58)

**Comment by DickenSerm** [comment]
Ah, take another port in Portainer's web editor (For Example: 3000):
```
   ports:
      - "3000:5000"
```
Then you have to adapt the URL with the new port in the webhook in Emby:
```
http://localhost:3000/webhook
```

Port 5000 is used by default for the DSM at Synology.

And the environment variable values ​​for
EMBY_BASE_URL
EMBY_API_KEY
TELEGRAM_BOT_TOKEN
TELEGRAM_CHAT_ID 
must be in ""

**Comment by Fahmula** [comment]
@spectre4094 Is the issue resolved?
